buildenv/docker/jdk11/s390x/ubuntu16/Dockerfile

```diff
@@ -39,7 +39,7 @@ RUN apt-get update \
     cpio \
     file \
     g++-7 \
-    gcc-7 \
+    gcc-7=7.4.0-1ubuntu1~16.04~ppa1 \
```

Comment:
Will 7.4 always be in the repo? If not I think we should pull the Adopt binary.

Reply:
I'm not sure. I don't see any other versions of gcc-7 in it, so I do not believe that It will stay.